docs/robots/jackal/user_manual_jackal.mdx

```diff
@@ -1,12 +1,358 @@
+
+### Onboard Computer Setup
+
+If you purchased an Onboard Computer from Clearpath Robotics with Jackal, it is already installed, connected, and powered.
+If not, or to set up your own Onboard Computer, see the [Jackal Tutorials](/docs/robots/jackal/tutorials_jackal#jackal-software-setup).
+
+Note that the Onboard Computer must be connected to the USB port on the Jackal MCU, which is accessible inside the Jackal.
+
+### Powering Up
+
+If you've just unpacked Jackal from its packaging, you may need to open it up and connect the battery.
+
+Press the power button <img src="/img/robot_images/dingo_images/icon-power.png" width="20" /> on Jackal's HMI panel.
+The LEDs should show a test pattern, after which you will wait about 30 seconds for the internal computer to finish booting up.
+Once it has finished booting, the Comms indicator <img src="/img/robot_images/dingo_images/icon-comms.png" width="20" /> should turn green.
+This indicates that ROS is up on the computer and has established communications with the Jackal base.
+
+### Network Configuration
+
+To get Jackal connected to your local Wi-Fi network, you must first access the Onboard Computer using a wired connection to the robot.
+Open the chassis, lower the computer tray, and connect to the network port labeled STATIC with a standard
+ethernet cable. See the [Jackal Tutorials](/docs/robots/jackal/tutorials_jackal#jackal-networking) for details
+on completing the Wi-Fi setup.
+
+### Connecting and Using the Controller {#controller}
+
+To use your controller, it must be paired with your Jackal and powered on.
+If your purchased the Onboard Computer from Clearpath Robotics, your controller is already paired with your Jackal.
+If your controller has become unpaired or if you wish to pair a new controller, following the instructions in
+the [Jackal Tutorials](/docs/robots/jackal/tutorials_jackal#pairing-the-controller).
+
+To teleoperate Jackal, first clear the motion-stop.
+Then, power on the controller by pressing the `PS` button on the controller.
+Once the blue LED on the top of the controller goes solid, you are paired and ready to drive.
+Hold the L1 trigger button (deadman switch) and push the left thumbstick forward to drive the Jackal.
+For full speed mode, hold the R1 trigger.
+See the figure below for the Sony PS4 controls layout.
+If you are not seeing any action, refer to [Support](#jackal-support) to get in touch with support.
+
+<center>
+  <figure>
+    <img
+      src="/img/robot_images/jackal_images/jackal-sony-ps4-controller-labelled.png"
+      width="600"
+    />
+    <figcaption>PS4 Controls Layout</figcaption>
+  </figure>
+</center>
+
+### Using ROS
+
+<ComponentUsingRos />
+
+### Battery Charging
+
+When finished with the Jackal, press and release the power button <img src="/img/robot_images/dingo_images/icon-power.png" width="20" /> to power it off.
+
+Jackal's Li-Ion battery pack may be charged internal to the chassis: simply plug in
```

Comment:
Suggest changing this to 'lithium battery'

Reply:
Updated